src/angular/components/send/add-edit.component.ts

```diff
@@ -178,6 +183,12 @@ export class AddEditComponent implements OnInit {
             }
         }
 
+        this.disableAnonymousSend = disableSendPolicies.some(p =>
+            p.data?.disableSend === DisableSendType.DisableAnonymous);
+        this.disableSend = (this.disableAnonymousSend && this.editMode && this.send.hideEmail) ||
```

Comment:
This logic doesn't make sense, either Send is disabled by policy or not, shouldn't make a difference whether anonymous is selected in this scenario.